packages/react-components/src/Concepts/Positioning/PositioningShorthandPositions.stories.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { PositionedComponent, useGridExampleStyles } from './utils.stories';
+
+export { Default } from './PositioningDefault.stories';
+
+export const ShorthandPositions = () => {
+  const styles = useGridExampleStyles();
+
+  return (
+    <>
+      <div className={styles.targetContainer}>
+        <div className={styles.instructions}>Hover over each box to see its positioned element</div>
+        <PositionedComponent
+          positioning="above-start"
+          targetClassName={styles.aboveStart}
+          targetContent="above-start ↑"
+        />
+        <PositionedComponent positioning="above" targetClassName={styles.above} targetContent="above ↑" />
+        <PositionedComponent positioning="above-end" targetClassName={styles.aboveEnd} targetContent="above-end ↑" />
```

Comment:
Let's improve this a bit, so arrows will show the actual position.

```suggestion
        <PositionedComponent
          positioning="above-start"
          targetClassName={styles.aboveStart}
          targetContent=" ↑ above-start"
        />
        <PositionedComponent positioning="above" targetClassName={styles.above} targetContent=" ↑ above ↑" />
        <PositionedComponent positioning="above-end" targetClassName={styles.aboveEnd} targetContent="above-end ↑" />
```